CodeStarterCamp_Week3/main.swift

```diff
@@ -24,22 +25,33 @@ class Person {
         print("제 이름은 \(name)이고 \(age)살 입니다. 소지금은 \(money)입니다.")
     }
     
-    func orderingCoffee(coffee: Coffee) {
-        print("\(coffee.rawValue)를 주문합니다.")
+    func orderCoffee(coffee: Coffee, coffeeShop: CoffeeShop) {
+        if money >= coffee.price {
+            print("\(name) 님이 \(coffee.rawValue)를 주문합니다.")
+            money = money - coffee.price
+            coffeeShop.makeCoffee(coffee: coffee, from: name)
+        } else {
+            print("잔액이 \(coffee.price - money)원만큼 부족합니다")
+        }
     }
```

Comment:
```suggestion
    func orderCoffee(_ coffee: Coffee, at coffeeShop: CoffeeShop) {
        if money >= coffee.price {
            print("\(name) 님이 \(coffee.rawValue)를 주문합니다.")
            money = money - coffee.price
            coffeeShop.makeCoffee(coffee: coffee, from: name)
        } else {
            print("잔액이 \(coffee.price - money)원만큼 부족합니다")
        }
    }
```

이렇게 전달인자 레이블을 활용해봐도 좋을 것 같아요. 문장형태로 만들어졌죠?